test/functional/test_framework/test_node.py

```diff
@@ -84,6 +84,13 @@ def __init__(self, i, datadir, rpchost, timewait, binary, stderr, mocktime, cove
 
         self.p2ps = []
 
+    def __del__(self):
+        # Ensure that we don't leave any bitcoind processes lying around after
+        # the test ends
+        if self.process:
+            self.log.info("Cleaning up leftover process")
```

Comment:
Looks like the log is flushed and closed when the tests are done, so wouldn't work at this point.

Reply:
Odd, I was able to see this logging work when I tested locally.  Will remove.

Reply:
Yes, it appears that logging *can* work after calling `logging.shutdown()`, but the documentation states that no calls to logging *should* be made after shutdown: https://docs.python.org/3/library/logging.html#logging.shutdown